src/components/DetailSidebar/DetailSidebar.tsx

```diff
@@ -5,7 +5,7 @@ import CardContent from '@material-ui/core/CardContent';
 import List from '@material-ui/core/List';
 
 import ActionBar from '../ActionBar/ActionBar';
-import Author from '../Author';
+// import Authors from '../Author';
```

Comment:
The author component is not rendered any more in this sidebar as it’s commented. Is this correct?

Reply:
Good catch! that is the reason I still like to have Snapshots...thank you :-)